src/site/content/en/blog/webcodecs/index.md

```diff
@@ -217,12 +253,17 @@ await encoder.flush();
 
 ### Decoding
 
+```html
+  <Img src="Decoding.png" alt="The path from the network or storage to a Canvas or an ImageBitmap." width="960" height="720" />
+```
```

Comment:
Are these supposed to be images shown in the article?

Reply:
yes, <figure> tag didn't work for me :(

Reply:
This won't work either. I have to upload them somewhere and give you the links. Can you please put the figure tag back in.

Your images have a lot of extra whitespace. Do you mind if I crop them?

Reply:
I brought back the figure tags. Feel free to crop or ever redraw images.